README.md

```diff
@@ -9,4 +9,4 @@ CDash for extensions: [http://slicer.cdash.org](http://slicer.cdash.org/index.ph
 
 For publications please include this text (modifying the initial part to describe your SlicerDMRI use case):
 
-"We performed diffusion MRI and/or tractography analysis and/or visualization in 3D Slicer (http://www.slicer.org) via the SlicerDMRI project (https://github.com/SlicerDMRI), funded by NIH U01 CA199459."
+"We performed diffusion MRI and/or tractography analysis and/or visualization in 3D Slicer (https://www.slicer.org) via the SlicerDMRI project (http://dmri.slicer.org/), funded by NIH U01 CA199459."
```

Comment:
Let's fix the deployment instead of reverting back to `http`

Reply:
This is related to https://github.com/SlicerDMRI/slicerdmri.github.io

Reply:
> Let's fix the deployment instead of reverting back to http

@jcfr I'm not reverting back to http: the previous link pointed to GitHub, which uses https: what I proposed is to point to the tool's webiste, which is http. Making the website transition to https would be ideal, but that is the topic of a separate PR.

Reply:
This has been fixed by enforcing https.

![image](https://github.com/SlicerDMRI/SlicerDMRI/assets/219043/6726ff82-4772-49f2-bd45-3dabb46efe7b)

Reply:
Thanks doing that. Done in 017a081.